analyzers/tests/SonarAnalyzer.UnitTest/Rules/OptionStrictOnTest.cs

```diff
@@ -0,0 +1,62 @@
+﻿/*
```

Comment:
Scaffolding: Update of file `VbRuleTypeMapping.cs` is missing. Can you add the changes to it from my original scaffolding branch?

It's a single line, so it might be easier to do it by hand.

Reply:
done.